acceptance/suites/pre_suite/compat_common/30_install_dev_repos.rb

```diff
@@ -1,4 +1,4 @@
-install_opts = options.merge({ :dev_builds_repos => ["PC1"]})
+install_opts = options.merge({ :dev_builds_repos => ["puppet"]})
```

Comment:
The only thing I'm unsure of here is how beaker handles this. I haven't looked closely at this method to make sure this will work